pkg/account/api/account_test.go

```diff
@@ -63,27 +64,28 @@ func TestCreateAccountV2MySQL(t *testing.T) {
 		req         *accountproto.CreateAccountV2Request
 		expectedErr error
 	}{
-		{
-			desc: "errNoCommand",
-			req: &accountproto.CreateAccountV2Request{
-				Command:        nil,
-				OrganizationId: "org0",
-			},
-			setup: func(s *AccountService) {
-				s.accountStorage.(*accstoragemock.MockAccountStorage).EXPECT().GetAccountV2(
-					gomock.Any(), gomock.Any(), gomock.Any(),
-				).Return(&domain.AccountV2{
-					AccountV2: &accountproto.AccountV2{
-						Email:            "bucketeer@example.com",
-						FirstName:        "Test",
-						LastName:         "User",
-						Language:         "en",
-						OrganizationRole: accountproto.AccountV2_Role_Organization_ADMIN,
-					},
-				}, nil)
-			},
-			expectedErr: createError(statusNoCommand, localizer.MustLocalizeWithTemplate(locale.RequiredFieldTemplate, "command")),
-		},
+		// deprecating command
```

Comment:
Yeah I have removed the unnecessary comments. Test case for create account v2 have already been implemented in this pr, please find TestCreateAccountV2NoCommandMySQL

Reply:
oh and because in enable, disable and delete account, the commands are just empty messages, so in test suites I just remove it in mock input

<img width="364" alt="image" src="https://github.com/user-attachments/assets/57af1a58-0bf6-4663-8cc6-c257502df9aa">